tests/cli/test_suite.py

```diff
@@ -3141,7 +3141,7 @@ def test_suite_new_profile_runs_notebook_no_jupyter(
     #     "random_seed": 43792,
     # },
     # numeric_columns_high_precision_rule={
-    #     "round_decimals": 4,   # parameters will be rounded to 4 decimal places for metrics that increase precision
+    #     "round_decimals": None,   # parameters will be rounded to the maximum number of decimal places for metrics that increase precision
```

Comment:
@NathanFarmer No, please let us leave it at 4 -- that was a good approach, meaningful for high precision.  Thank you.